keys/errors.go

```diff
@@ -25,6 +25,10 @@ var (
 	ErrPrivKeyUndecodable   = errors.New("could not decode privkey")
 	ErrPrivKeyLengthInvalid = errors.New("invalid privkey length")
 
+	ErrKeyGenSecp256k1Failed    = errors.New("keygen: new private key")
```

Comment:
nit: error message can be rephrased to be more relevant to the actual error
```suggestion
	ErrKeyGenSecp256k1Failed    = errors.New("keygen: error generating keypair for secp256k1 curve type")
```

Reply:
Agree with @qiwu7's nit here!

Reply:
👍